cmd/config.go

```diff
@@ -0,0 +1,103 @@
+/*
+ * JuiceFS, Copyright (C) 2021 Juicedata, Inc.
+ *
+ * This program is free software: you can use, redistribute, and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3
+ * or later ("AGPL"), as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/juicedata/juicefs/pkg/meta"
+	"github.com/urfave/cli/v2"
+)
+
+var changableConfigs = []cli.Flag{
+	&cli.Uint64Flag{
+		Name:  "capacity",
+		Value: 0,
+		Usage: "the limit for space in GiB",
+	},
+	&cli.Uint64Flag{
+		Name:  "inodes",
+		Value: 0,
+		Usage: "the limit for number of inodes",
+	},
+	&cli.StringFlag{
+		Name:  "access-key",
+		Usage: "Access key for object storage (env ACCESS_KEY)",
+	},
+	&cli.StringFlag{
+		Name:  "secret-key",
+		Usage: "Secret key for object storage (env SECRET_KEY)",
+	},
+	&cli.IntFlag{
+		Name:  "trash-days",
+		Value: 1,
+		Usage: "number of days after which removed files will be permanently deleted",
+	},
+}
+
+func config(ctx *cli.Context) error {
+	setLoggerLevel(ctx)
+	if ctx.Args().Len() < 1 {
+		return fmt.Errorf("META-URL is needed")
+	}
+	m := meta.NewClient(ctx.Args().Get(0), &meta.Config{Retries: 10, Strict: true})
+
+	format, err := m.Load()
+	if err != nil {
+		return err
+	}
+	var msg strings.Builder
+	for _, flag := range ctx.LocalFlagNames() {
+		switch flag {
```

Comment:
ignore the default value

Reply:
`LocalFlagNames` returns only flags explicitly specified in the command